entrypoints/injected/components/Header.tsx

```diff
@@ -59,7 +59,14 @@ const Header = forwardRef(function (
       canvas?.removeEventListener('wheel', wheelHandler, false)
     }
   }, [altPressing, metaPressing])
-
+  // The DropdownMenuTrigger has been disabled from opening the menu when holding down the Ctrl key
+  // https://github.com/radix-ui/primitives/blob/main/packages/react/dropdown-menu/src/DropdownMenu.tsx#L119
+  const onMouseEnter = function () {
```

Comment:
Should we wrapper the fns with `useCallback` ?

Reply:
done